vividus-studio-server/vividus-studio-plugin/src/main/java/org/vividus/studio/plugin/service/StepDefinitionResolver.java

```diff
@@ -55,7 +55,8 @@ public class StepDefinitionResolver implements IStepDefinitionsAware, StepDefini
         "Scenario:",
         "Meta:",
         "GivenStories:",
-        "Examples:"
+        "Examples:",
+        "Composite:"
```

Comment:
why is keyword from composite steps file put into the list of story file keywords?